docs/pages/blog/react-19-update.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Migrating MUI X to React 19
+description: How we migrated to React 19 while maintaining backward compatibility.
```

Comment:
If you update the title to include some of this text, you could rephrase and expand on it like this in the description:

```suggestion
description: Learn how we migrated our library's codebase to React 19 while maintaining backward compatibility.
```